dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ConstantPoolMethodref.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
+
+/**
+ * A representation of an Methodref entry in constant pool.
+ * In Java class file ConstantPool section,
+ * a methodref entry has a pointer (index) of the actual value of class name, method name and its
+ * type information
+ * On the other hand, this class holds these values store directly so that we can read them easily
+ */
+class ConstantPoolMethodref {
```

Comment:
This class seems to just be `MethodAndSignature` (which I recommend naming `MethodSignature` in another comment) plus the fully-qualified class name.

Since this class doesn't store indexes, I don't think `Methodref` is a good way to describe this.

I feel like maybe this class should be named `FullyQualifiedMethodSignature`, and should hold a String for the fully-qualified class name and a `MethodSignature` instance for the method name + parameter list.